custom/panel_templates/Default/core/avatars.tpl

```diff
@@ -155,6 +152,45 @@
     <!-- End Wrapper -->
 </div>
 
+<script>
+
+const perspective_selector = document.getElementById('inputAvatarPerspective');
```

Comment:
Just a side note, should we officially drop support for IE and show some form of banner to IE users, asking them to switch browsers?

Reply:
yeah, i agree. i'll make an issue now so we can remember that